app2.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+var firstAndPike = {
+  minCust: 23,
+  maxCust: 65,
+  avgCookiePerSale: 6.3
+};
+
+var seaTacAir = {
+  minCust: 3,
+  maxCust: 24,
+  avgCookiePerSale: 1.2
+};
+
+var seattleCenter = {
+  minCust: 11,
+  maxCust: 38,
+  avgCookiePerSale: 3.7
+};
+
+var capHill = {
+  minCust: 20,
+  maxCust: 38,
+  avgCookiePerSale: 2.3
+};
+
+var alki = {
+  minCust: 2,
+  maxCust: 16,
+  avgCookiePerSale: 4.6
+};
+
+/*
+var storesMinMax = [
```

Comment:
get rid of commented out code, if you want to be able to look back on it, just make another commit.